Automattic/woocommerce-payments
# Revert "Remove ' in front of negative values on csv files"

Part of #3333

#### Changes proposed in this Pull Request
This PR reverts #2940 temporarily. The revert is necessary because a bug that make the WooCommerce > Home page blank after WooCommerce 5.9.0 release. 

#### Testing instructions
- With WooCommerce and WCPay plugins installed and active
- Go to WooCommerce > Home
- Make sure the page looks like this
<img width="747" alt="Screen Shot 2021-11-12 at 12 20 48 PM" src="https://user-images.githubusercontent.com/6342964/141490601-03083041-9f31-4669-a425-2670b948a7a2.png">
